packages/frontend-shared/cypress/e2e/support/e2eSupport.ts

```diff
@@ -193,11 +197,45 @@ function startAppServer (mode: 'component' | 'e2e' = 'e2e') {
   return logInternal('startAppServer', (log) => {
     return cy.withCtx(async (ctx, o) => {
       ctx.actions.project.setCurrentTestingType(o.mode)
-      // ctx.lifecycleManager.isReady()
-      await ctx.actions.project.initializeActiveProject({
-        skipPluginInitializeForTesting: true,
+      const isInitialized = o.pDefer()
+      const initializeActive = ctx.actions.project.initializeActiveProject
+      const onErrorStub = o.sinon.stub(ctx, 'onError')
+      // @ts-expect-error - errors b/c it's a private method
+      const onLoadErrorStub = o.sinon.stub(ctx.lifecycleManager, 'onLoadError')
+      const initializeActiveProjectStub = o.sinon.stub(ctx.actions.project, 'initializeActiveProject')
+
+      function restoreStubs () {
+        onErrorStub.restore()
+        onLoadErrorStub.restore()
+        initializeActiveProjectStub.restore()
+      }
+
+      function onStartAppError (e: Error) {
+        isInitialized.reject(e)
+        restoreStubs()
+      }
+
+      onErrorStub.callsFake(onStartAppError)
+      onLoadErrorStub.callsFake(onStartAppError)
+
+      initializeActiveProjectStub.callsFake(async function (this: any, ...args) {
+        try {
+          const result = await initializeActive.apply(this, args)
+
+          isInitialized.resolve(result)
+
+          return result
+        } catch (e) {
+          isInitialized.reject(e)
+        } finally {
+          restoreStubs()
+        }
```

Comment:
Used to capture any error that occurs which blocks the server from starting, which we'd otherwise see in the launchpad flow, but we need to surface as an error for the Cypress command since we don't have a launchpad in e2e tests